enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -318,10 +318,12 @@ def total_spend_limit_for_all_policies_associated_to_subsidy(self):
         Sums the policies spend_limit excluding the db's instance of this policy
         """
         policy_balances = []
-        policy_balances.append(self.spend_limit or 0)
+        if self.active:
+            policy_balances.append(self.spend_limit or 0)
```

Comment:
Nested this logic in the conditional to check to see if the policy we are modifying is not active, we do not want to include it in the policy balances. This is in parity with the changes to the filtering that only looks for active sibling policies.